README.md

```diff
@@ -98,21 +105,15 @@ If you are developing a ticket that deals with any sort of monitoring or asynchr
 BACK_RAWLS=true ./config/docker-rsync-local-rawls.sh
 ```
 
-**Important**: It is highly recommended that use your own Cloud SQL instance when running an instance of back Rawls by cloning the [dev Rawls DB](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/overview?project=broad-dsde-dev). See note below on database work.
-
-#### Developing Database Schema Changes
-
-If you are writing Liquibase migrations or doing database work, it is mandatory that you create your own Cloud SQL instance and develop against that. To point to your own CloudSQL instance, modify `/config/sqlproxy.env` accordingly.
-
+### Developing Database Schema Changes
 
+If you are writing Liquibase migrations or doing database work, set up a database clone by following the steps for [running Back Rawls](#additional-requirements-for-back-rawls).
```

Comment:
This might be a bit confusing if we already tell the dev to use a clone. It might make sense to move the justification of using a clone up a bit